Makefile

```diff
@@ -21,7 +21,7 @@ help:
 	@echo "HELP              Set to true to print the help message (default: false) e.g. <make run HELP=true>"
 	@echo "PRINT_BATCH_SIZE  Set the batch size for printing the results e.g. <make run PRINT_BATCH_SIZE=100>"
 	@echo "TOTAL_TIME        Set the total time for the simulation e.g. <make run TOTAL_TIME=1000>"
-	@echo "DATA_DIR		  	 Set the data directory for the simulation containing simulation files and results e.g. <make run DATA_DIR=/data>"
+	@echo "DATA_DIR		  	 Set the data directory for the simulation containing simulation files and results e.g. <make run-docker DATA_DIR="/Users/anon/data>""
```

Comment:
Spacing looks off here?

Reply:
hmm.... spacing looks fine on local.

Reply:
Looks like you're using tabs and the rest are spaces - can manually replace with spaces or set your editor to convert tabs to spaces.

Reply:
fixed.